Lux/src/warp_field.cpp

```diff
@@ -0,0 +1,18 @@
+#include "image.hpp"
+#include "vector_field.hpp"
+
```

Comment:
a file named `warp_field.cpp` should include `warp_field.hpp` as the first include. this avoids "missing dependency" errors.

Reply:
Committed a complete warp_field implementation. If there are no further issues I can resolve both conversations and merge.